packages/cli-tool/src/template/initialize-vite-app.ts

```diff
@@ -0,0 +1,85 @@
+import { CliUx } from '@oclif/core';
+
+import runCommand from '../helpers/child-process';
+import { replaceLine } from '../helpers/file-editor';
+
+const TEMPLATE_OWNER = 'nimblehq';
+const TEMPLATE_REPO = 'react-templates';
+const replaceAppNameInFiles = ['package.json', 'index.html'];
+
+type InitViteOptions = {
+  dest: string;
+  appName: string;
+  branch: string;
+};
+
+const downloadTemplateRepository = (
```

Comment:
Should we move these functions to a helper? Then we can re-use them later as well. Just need to pass an option object, and get the expected folder

Reply:
272e58f ✅ 

Please help me to review if that fits what you had in mind. I don't really like the dependence with the run-command (and so the need for the `dest` argument)... but I think it is hard to do it differently 🤔